bpm/src/main/java/org/jboss/pnc/bpm/ConnectorSelector.java

```diff
@@ -35,9 +35,14 @@ public static boolean useNewProcess(BpmTask task) {
             Map<String, String> genericParameters = buildTask.getBuildTask()
                     .getBuildConfigurationAudited()
                     .getGenericParameters();
-            if (true || genericParameters.getOrDefault(GENERIC_PARAMETER_KEY, "").equals(RHPAM)) {
-                return true;
-            }
+            return useNewProcessForBuild(genericParameters);
+        }
+        return false;
+    }
+
+    public static boolean useNewProcessForBuild(Map<String, String> genericParameters) {
+        if (true || genericParameters.getOrDefault(GENERIC_PARAMETER_KEY, "").equals(RHPAM)) {
+            return true;
```

Comment:
you don't need if, you can just return the value of the condition;

Reply:
BTW there is plan to change it in the future to not always return true?

Reply:
"true or" is there for easy roll-back if we need it, at some point we have to clean it up.